addOns/automation/src/main/java/org/zaproxy/addon/automation/jobs/SpiderJob.java

```diff
@@ -51,15 +52,10 @@
 
     private static final String PARAM_CONTEXT = "context";
     private static final String PARAM_URL = "url";
-    private static final String PARAM_FAIL_IF_LESS_URLS = "failIfFoundUrlsLessThan";
-    private static final String PARAM_WARN_IF_LESS_URLS = "warnIfFoundUrlsLessThan";
```

Comment:
Should we keep these (for now), but warn if they are used and not actually use them?
In theory people could already be using them which would mean their jobs would start to fail. Warning on then would mean the jobs carry on working unless they are set to fail-on-warn.

Reply:
Okay, we decided on IRC earlier to remove them, but I'll add them back again.

Reply:
I'm also ok with that, is was really a question :)